src/main/kotlin/cc/hyperium/installer/HyperiumInstaller.kt

```diff
@@ -28,7 +28,7 @@ class InstallerApp : App(InstallerView::class, InstallerStyles::class)
 
 fun main(args: Array<String>) {
     // Update to current date when changing version
-    Installer.logger.info("Running Hyperium installer version 1.5.1/March 18th 2020")
+    Installer.logger.info("Running Hyperium installer version 1.5.2/March 22nd 2020")
```

Comment:
can we not hardcode this and somehow have gradle inject it

Reply:
no fuck you

Reply:
have fun doing that cube. It's your shit code